misc/tremulous-grangerhub.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=GrangerHub Tremulous
+Comment=A free team based FPS/RTS hybrid built on the ioq3 engine. Includes GrangerHub community updates.
```

```suggestion
Comment=A free team based FPS/RTS hybrid built on the ioq3 engine. Includes GrangerHub community updates
```

Just like with the AppStream metadata summary, there should not be a dot at the end.

---

Also, it should ideally match the AppStream summary text.